operator/pkg/translate/icp_iop.go

```diff
@@ -33,7 +36,17 @@ kind: IstioOperator
 
 // ReadTranslations reads a file at filePath with key:value pairs in the format expected by TranslateICPToIOP.
 func ReadTranslations(filePath string) (map[string]string, error) {
-	b, err := ioutil.ReadFile(filePath)
+	return readTranslationsWithFunc(filePath, ioutil.ReadFile)
+}
+
+// ReadTranslationsVFS reads a file at filePath in VFS with key:value pairs in the format expected by TranslateICPToIOP.
+func ReadTranslationsVFS(filePath string) (map[string]string, error) {
```

Comment:
we seem to be doing the same thing in our parallel pr's - i'd argue in favor of the approach in my pr since vfs is the normal case so exposing vfs in the name through this package feels like leaking an implementation detail.
but obviously i am biased so lmk if you feel otherwise:
https://github.com/istio/istio/pull/20333/files#diff-5f568b6f75a882ebd99923fb0e3e315eR36

Reply:
Great! I will rename first, and see if we can merge them depends on the order of commits